app/src/test/java/edu/wpi/first/shuffleboard/app/json/DashboardTabPaneSaverTest.java

```diff
@@ -29,15 +29,21 @@ public void setUp() {
     NetworkTableUtils.shutdown();
     NetworkTablesJNI.setUpdateRate(0.01);
     AsyncUtils.setAsyncRunner(Runnable::run);
-    NetworkTablesJNI.putDouble("/LiveWindow/TestSystem/Ultrasonic/Value", 0.5);
-    NetworkTablesJNI.putDouble("/LiveWindow/TestSystem/Compass/Value", 0.5);
-    NetworkTablesJNI.putDouble("/LiveWindow/Elevator/p", 0.5);
-    NetworkTablesJNI.putDouble("/LiveWindow/Elevator/d", 0.5);
-    NetworkTablesJNI.putDouble("/LiveWindow/Elevator/f", 0.5);
-
-    NetworkTablesJNI.putBoolean("/LiveWindow/~STATUS~/LW Enabled", true);
-    NetworkTablesJNI.putBoolean("/LiveWindow/Elevator/enabled", false);
-    NetworkTableUtils.waitForNtcoreEvents();
+    NetworkTableUtils.waitForNtcoreEvents(()
+        -> NetworkTablesJNI.putDouble("/LiveWindow/TestSystem/Ultrasonic/Value", 0.5));
+    NetworkTableUtils.waitForNtcoreEvents(()
+        -> NetworkTablesJNI.putDouble("/LiveWindow/TestSystem/Compass/Value", 0.5));
+    NetworkTableUtils.waitForNtcoreEvents(()
+        -> NetworkTablesJNI.putDouble("/LiveWindow/Elevator/p", 0.5));
+    NetworkTableUtils.waitForNtcoreEvents(()
+        -> NetworkTablesJNI.putDouble("/LiveWindow/Elevator/d", 0.5));
+    NetworkTableUtils.waitForNtcoreEvents(()
+        -> NetworkTablesJNI.putDouble("/LiveWindow/Elevator/f", 0.5));
+
+    NetworkTableUtils.waitForNtcoreEvents(()
+        -> NetworkTablesJNI.putBoolean("/LiveWindow/~STATUS~/LW Enabled", true));
+    NetworkTableUtils.waitForNtcoreEvents(()
+        -> NetworkTablesJNI.putBoolean("/LiveWindow/Elevator/enabled", false));
```

Comment:
same comment as the one preceding this one (https://en.wikipedia.org/wiki/Ibid.)